fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/BindVisitor.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query;
+
+import java.math.BigDecimal;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.time.Instant;
+import java.util.Calendar;
+import java.util.TimeZone;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+import com.ibm.fhir.database.utils.query.expression.BindMarkerNodeVisitor;
+
+/**
+ * Visitor implementation to bind (set) values for a PreparedStatement. The bindXX
```

Comment:
Could we split this to two separate paragraphs? it reads a bit easier.

Reply:
ok